src/webgl/loading.js

```diff
@@ -96,29 +98,33 @@ import './p5.Geometry';
  * @param  {String} path
  * @param  {function(p5.Geometry)} [successCallback]
  * @param  {function(Event)} [failureCallback]
+ * @param  {String} [fileType]
  * @return {p5.Geometry} the <a href="#/p5.Geometry">p5.Geometry</a> object
  */
 p5.prototype.loadModel = function(path) {
   p5._validateParameters('loadModel', arguments);
   let normalize;
   let successCallback;
   let failureCallback;
+  let fileType;
   if (typeof arguments[1] === 'boolean') {
     normalize = arguments[1];
     successCallback = arguments[2];
     failureCallback = arguments[3];
+    fileType = arguments[4];
   } else {
     normalize = false;
     successCallback = arguments[1];
     failureCallback = arguments[2];
+    fileType = arguments[3];
   }
 
-  const fileType = path.slice(-4);
+  fileType = fileType == null ? path.slice(-4) : fileType;
```

Comment:
can you just set this on line 109, then it gets overwritten if specified by the if statements below?

Reply:
Thanks. I moved the default definition on line 109 and if `fileType` is given, it will overwrite  the default one.